styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteRunner.java

```diff
@@ -43,6 +45,12 @@ static FlyteRunner noop() {
     return new NoopFlyteRunner();
   }
 
+  static FlyteRunner flyteAdmin(final String runnerId,
```

Comment:
should we call this create?

Reply:
actually there is a create method in the FlyteAdminClient, can we refactor to use it as well? https://github.com/spotify/styx/blob/master/styx-flyte-client/src/main/java/com/spotify/styx/flyte/client/FlyteAdminClient.java#L51

Reply:
`FlyteRunner` contains several factory methods: `flyteAdmin`, `noop` and `routing`, each of them returning a different class

Reply:
i see 👍

Reply:
> actually there is a create method in the FlyteAdminClient, can we refactor to use it as well? https://github.com/spotify/styx/blob/master/styx-flyte-client/src/main/java/com/spotify/styx/flyte/client/FlyteAdminClient.java#L51

I did in the next commit